test/html/normalize.vows.js

```diff
@@ -85,5 +85,11 @@ wows.describe('html normalizer').addBatch({
     	
     	var dom = new DOMParser().parseFromString('<textarea>alert(a<b&&c?"<br>":">>");</textarea>','text/xml');
     	assert(dom+'', '<textarea>alert(a&lt;b&amp;&amp;c?"<br/>":">>");</textarea>');
+	},
+    'Euro characters': function () {
```

Comment:
```suggestion
    'European entities': function () {
```

I don't know if it is worth making this change or not:)